frontend/www/js/omegaup/components/course/StudentProgress.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <tr>
+    <td class="text-center align-middle">
+      <a v-bind:href="studentProgressUrl">
+        {{ student.name || student.username }}
+      </a>
+    </td>
+    <td
+      class="score flex-column justify-content-center align-items-center"
+      v-for="assignment in assignments"
+    >
+      <p class="mb-1 text-center">{{ Math.round(score(assignment)) }}%</p>
+      <div class="d-flex justify-content-center">
+        <div v-if="student.progress.hasOwnProperty(assignment.alias) == false">
+          {{ T.wordsProblemsUnsolved }}
+        </div>
+        <div v-else class="d-flex border border-dark">
+          <div
+            v-for="problemScore in student.progress[assignment.alias]"
+            v-bind:class="getProblemColor(Math.round(problemScore))"
+            data-toggle="tooltip"
+            data-placement="bottom"
+            v-bind:title="Math.round(problemScore) + '%'"
```

Comment:
nit: the use of [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) is preferred over concatenation with `+`:

```suggestion
            v-bind:title="`{Math.round(problemScore)}%`"
```

Reply:
When I do that, I get this:
![image](https://user-images.githubusercontent.com/5478067/89112273-4aa5c100-d415-11ea-8c33-e31812ba272a.png)
Am I doing that wrong? I thought you couldn't do [template literals inside attributes](https://vuejs.org/v2/guide/syntax.html#Attributes)

Reply:
Change that to:
```js
v-bind:title="`${Math.round(problemScore)}%`"
```
and it'll work

Reply:
errr. yes, sorry, i missed a `$` ^^;;